docs/content/tutorial_code/deploying_with_gloo.md

```diff
@@ -0,0 +1,191 @@
+---
+title: "Deploying Filters to Gloo"
+weight: 2
+description: Deploy a wasm filter using Gloo as the control plane.
+---
+
+In this tutorial we'll deploy an existing WebAssembly (WASM) module from [the WebAssembly Hub](https://webassemblyhub.io) directly to Envoy via [Gloo](https://docs.solo.io/gloo/latest) installed to our kubernetes cluster.
+
+## Prepare environment
+
+To get started, let's deploy a sample service that we can call through Envoy. We'll deploy the sample petstore API:
+
+```shell
+kubectl apply -f \
+https://raw.githubusercontent.com/solo-io/gloo/master/example/petstore/petstore.yaml
+```
+
+You should now have the petstore running:
+
+```shell
+kubectl get po 
+```
+
+```
+
+NAME                        READY   STATUS    RESTARTS   AGE
+petstore-5dcf5d6b66-n8tjt   1/1     Running   0          2m20s
+```
+
+### Deploying Envoy
+
+In this tutorial, we'll use Gloo, an API Gateway based on Envoy that has built-in wasm support but these steps should also work for base Envoy.
+
+First, install Gloo via the helm chart:
+
+```shell
+helm repo update
+kubectl create ns gloo-system
+helm install gloo-gateway gloo/gloo --namespace gloo-system \
+  --set global.wasm.enabled=true
+```
```

Comment:
Helm 2 or 3? or both?

Reply:
tested with helm 3. compatibility depends on the gloo chart; is it compatible with helm 2?

Reply:
I guess I just meant we should say explicitly which one it is compatible with, since the ecosystem is in transition